pkg/cmd/pipeline/pipeline.go

```diff
@@ -12,38 +12,36 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package cmd
+package pipeline
 
 import (
 	"os"
 
 	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/cli"
+	"github.com/tektoncd/cli/pkg/flags"
 )
 
-var (
-	namespace string
-)
+func Command(p cli.Params) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "pipeline",
+		Aliases: []string{"p", "pipelines"},
+		Short:   "Manage pipelines",
 
-// pipelinesCmd represents the pipelines command
-var pipelinesCmd = &cobra.Command{
-	Use:     "pipelines",
-	Aliases: []string{"p", "pipeline"},
-	Short:   "Manage pipelines",
-	Run: func(cmd *cobra.Command, args []string) {
-		if len(args) == 0 {
-			cmd.Help()
-			os.Exit(0)
-		}
-	},
-}
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return flags.InitParams(p, cmd)
+		},
 
-func init() {
-	rootCmd.AddCommand(pipelinesCmd)
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				cmd.Help()
+				os.Exit(1)
```

Comment:
How about returning an `error` with an appropriate message rather calling `Exit()`,
if [root cmd](https://github.com/tektoncd/cli/pull/17/files#diff-6ce6b47ced67718ed4842ac0510704c5R31) calling exit function on `error` and `error` returned from this function could be cascade till root cmd `Executions()` ?

Reply:
+1 .. will do

Reply:
done!